docs/resources/object_storage_access_keys.md

```diff
@@ -0,0 +1,31 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "fastly_object_storage_access_keys Resource - terraform-provider-fastly"
+subcategory: ""
+description: |-
+  
+---
+
+# fastly_object_storage_access_keys (Resource)
+
+
```

Comment:
We should consider either duplicating some of [this](https://www.fastly.com/documentation/reference/api/services/resources/object-storage-access-keys/) content here, or at least including a link to that page.